packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -563,8 +563,12 @@ function parseBindings(
     }
 
     if (appDom.isQuery(elm)) {
+      let kind: 'query' | 'action' = 'query';
```

Comment:
Understood, I added "action" as a way to progressively perhaps do away with mutation in the code and eventually create a separate node type called "action", since if that is what we want to call it in the UI, should it also not be what we describe it in the codebase for simplicity?

Reply:
Personally I prefer either one way or the other:
* either we call it "action" everywhere and just do a translation to "mutation", right before we read/write to a file
* or we call it "mutation" everywhere and name it "action" just right before we display in the UI

The reason is that this prevents us from mismatching and constant translation all around the codebase. Ideally if the rename is done in a large PR, you start with the latter, just to avoid introducing many unnecessary chamges to unrelated code. Then in a separate PR update the codebase to the former.